test/unit/ajax.js

```diff
@@ -1197,6 +1197,25 @@ module( "ajax", {
 		{
 			url: url("data/json.php"),
 			converters: {
+				"text json": function( data ) {
+					ok( true, "promise converter called" );
+					return jQuery.Deferred().resolveWith( data );
```

Comment:
I want coverage to be "run all existing tests with a sync `"text json"` converter, and with an async `"text json"` converter." Any recommendation as to how to do that?